examples/java/com/google/census/CensusRunner.java

```diff
@@ -36,11 +39,42 @@
 
   public static void main(String args[]) {
     System.out.println("Hello Census World");
-    System.out.println("Default Tags: " + Census.getCensusContextFactory().getDefault());
```

Comment:
Nit: hard to see the changes due to extracting methods. Presumably biggest change here is catch blocks